src/main.js

```diff
@@ -4,14 +4,15 @@ var pastActivities = [];
 var studyButton = document.querySelector('.study');
 var meditateButton = document.querySelector('.meditate');
 var exerciseButton = document.querySelector('.exercise');
+var catButtons = document.querySelectorAll('.cat-button')
 var startButton = document.querySelector('.start-button');
 var startActivityBtn = document.querySelector('.start-activity-button');
 var logButton = document.getElementById('log-button');
 var createNewActivityBtn = document.querySelector('.new-activity-button');
 var accomplishInput = document.querySelector('.accomplish-input');
 var minutesInput = document.querySelector('.minutes-input');
 var secondsInput = document.querySelector('.seconds-input');
-var buttonText = document.querySelectorAll('.cat-titles')
+//var buttonText = document.querySelectorAll('.cat-titles')
```

Comment:
Also name convention, since it's more than one text content, we should use plural

Reply:
If I don't see cats on this page I am going to be deeply saddened...